LayoutTests/http/tests/links/resources/redirect-helper.pl

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/perl
+# Script to generate a 30x HTTP redirect (determined by the query parameter)
```

Comment:
It might be better to use a python script since we are moving out of php/perl, see for instance LayoutTests/http/tests/xmlhttprequest/resources/redirect-cors.py